src/components/TopNav/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Button = ({ onClick, className, children }) => {
+  return (
+    <div
```

Comment:
Rule: Semantic HTML over anything.

Reply:
thanks for pointing it out. `rafce` spoiled me 😂